common/net/http/withBody.go

```diff
@@ -199,3 +201,15 @@ func fields(errs validator.ValidationErrors, trans ut.Translator) common.FieldVa
 
 	return nil
 }
+
+func fieldsRequired(myMap common.FieldValidations) common.FieldValidations {
+	result := make(common.FieldValidations)
+
+	for key, value := range myMap {
```

Comment:
check myMap to avoid panic 
```suggestion
    if myMap == nil {
		return result
	}
	
	for key, value := range myMap {
```